components/automate-ui/src/styles/chef-theme.scss

```diff
@@ -19,7 +20,29 @@ $chef-warn:    mat-palette($chef-mat-warn);
 // Create the theme object (a Sass map containing all of the palettes).
 $chef-theme: mat-light-theme($chef-primary, $chef-accent, $chef-warn);
 
+// Define a custom typography config that overrides the font-family as well as the
+// `headlines` and `body-1` levels.
```

Comment:
Could you educate me a bit on what this does and why it is needed?

Reply:
@msorens this allows us to apply custom typography across the material components, I was asked to use Muli by @susanev 
https://material.angular.io/guide/typography